src/hidimstat/base_variable_importance.py

```diff
@@ -33,25 +33,31 @@ def __init__(self):
         super().__init__()
         self.importances_ = None
         self.pvalues_ = None
-        self.selections_ = None
+
+    def _check_importance(self):
+        """
+        Checks if the importance scores and p-values have been computed.
+        """
+        if self.importances_ is None:
+            raise ValueError(
+                "The importances need to be called before calling this method"
+            )
 
     def selection(
```

Comment:
To simplify this function, instead of having an `if arg is not None ..., else mask_arg = np.ones(...)`, why not have a unique mask variable? 
Unless I'm mistaken, the function is not meant to be used with multiple non-None arguments?